src/scss/constants.scss

```diff
@@ -148,6 +148,7 @@ $z-layers: (
   map-room: 4,
   map-grid: 3,
   map-room--unclickable: 2,
+  map-room--iframe: 4,
   map-background: 1,
```

Comment:
We aim to keep the order here (within each 'section') based on the `z-index` 'level'. So if we leave `map-room--iframe` as `4` then we should move it up next to `map-room`.

Though I wonder, should we have it set to `4`? Or should we have it set to `2`, the same as 'unclickable'? Or something entirely new (potentially bumping other layers 'up a number' to make room for it. When I originally envisaged this layering for the map, my intention was that we would have each item on it's own layer, so that we can explicitly control the positioning (rather than 'natural layering' occuring when multiple items of different 'types' on the same layer interact)

@margulies @sofisparkle Do you have any thoughts on what the layering here should be?

Reply:
I think it would be better to show this above the `map-grid`, but could shift `map-room` and everything higher up by 1

Reply:
Yeah, above the `map-grid` definitely makes sense to me as well, since it's more 'content you want to see' rather than 'part of the background' (like `unclickable` was intended as).

Pushing the other `z-index`s 'up by 1' to 'make room' seems like a good choice here.

Reply:
Changed the order in https://github.com/sparkletown/sparkle/pull/1427/commits/998a0612be928ccb5278cc3cb058ef0399f292e7